makefile

```diff
@@ -121,7 +121,7 @@ $(buildDir)/output.%.coverage: .FORCE
 $(buildDir)/output.%.coverage.html:$(buildDir)/output.%.coverage
 	go tool cover -html=$< -o $@
 $(buildDir)/output.smoke.test: $(buildDir)/$(name)
-	./$< &
+	./$< --localPath $(buildDir) &
```

Comment:
Does it make sense to use _bucketdata for this, or possibly a subfolder under buildDir? I'd be worried about writing to the build directory itself.